microscopium/screens/__init__.py

```diff
@@ -1,8 +1,11 @@
 from __future__ import absolute_import
-from . import myofusion
+from . import myofusion, cellomics
 
-__all__ = ['myofusion']
+__all__ = ['myofusion', 'cellomics']
 
-d = {'myofusion': {'index': myofusion.filename2coord,
-                   'fmap':  myofusion.feature_map}}
+d = {'myofusion': {'index': myofusion.filename2id,
+                   'fmap':  myofusion.feature_map},
```

Comment:
Just a thought.. wondering if "Myofusion" should change to "Definiens" seeing as these functions should be based on the platform where the images were obtained. Maybe a super-low priority issue to think about if we come across another Definiens dataset..?